src/AudioAnalysisTools/Events/Types/CompositeEvent.cs

```diff
@@ -57,6 +57,19 @@ public IEnumerable<ITrack> Tracks
             }
         }
 
+        public new double ScoreNormalised()
+        {
+            double sum = 0.0;
+            foreach (var @event in this.ComponentEvents)
+            {
+                sum += @event.Score;
+            }
+
+            var score = sum / this.ComponentEvents.Count;
+
+            return score / this.ScoreRange.Maximum;
+        }
+
```

Comment:
```suggestion
        public override double ScoreNormalised => this.ComponentEvents.Average()
```

1. Should be override
2. Should be property
3. Why not use score normalized from component events?
4. Using Score will almost certainly calculate an incorrect result if the range maximum's are different